public/src/components/changepassword/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import { ForgotPasswordRequest } from "../../../proto_build/actions/ForgotPassword_pb";
+import { ChangePasswordRequest } from "../../../proto_build/actions/ChangePassword_pb";
+import { DalalActionService } from "../../../proto_build/DalalMessage_pb_service";
+
+export interface FormState {
+
+    error: string | null,
+    disabled: boolean,
+    temp_password: string,
```

Comment:
variables are named in lower camel case in React. So change the names here accordingly.